### PR TITLE
Feature/penguins

### PR DESCRIPTION
**Pull request by AymenFJA**
This PR introduces a new use case support for the ICEBERG-Middleware (Penguins). The package will install RADICAL Ensemble Toolkit. To execute the penguins use case using the following command:

```
iceberg --resource xsede.bridges --queue RM --cpus 28 --gpus 0 --input_path ~/All_sites_images/A --output_path ~/pengins_output --walltime 30 penguins
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ export the following variables with appropriate values:
 export RADICAL_PILOT_DBURL=mongodb://usernane:password@mongodb_hostname:port/db_name
 export RMQ_ENDPOINT=ip_value
 export RMQ_PORT=port_number
-export VE_SEALS=ve_path/ve_name
+export RMQ_PASSWD=password
```
Comment (iparask) [inline README.md:40]: Can you add `RMQ_UNAME` as well in the documentation?

Comment (AymenFJA) [reply README.md:40]: Got it. I Will update this ASAP.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ export the following variables with appropriate values:
 export RADICAL_PILOT_DBURL=mongodb://usernane:password@mongodb_hostname:port/db_name
 export RMQ_ENDPOINT=ip_value
 export RMQ_PORT=port_number
+export RMQ_USERNAME=username
```
Comment (AymenFJA) [inline README.md:39]: @iparask documentation is updated now.

File: src/iceberg/executor/penguins.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                 model_path=None, epoch=None):
+
+        super(Penguins, self).__init__(name=name,
+                                    resource=resources['resource'],
+                                    queue=resources['queue'],
+                                    walltime=resources['walltime'],
+                                    cpus=resources['cpus'],
+                                    gpus=resources['gpus'],
+                                    project=project)
+        self._gpu_ids = gpu_ids
```
Comment (iparask) [inline src/iceberg/executor/penguins.py:39]: @AymenFJA what is `gpu_ids`? Do you want to force to use specific GPUs? What will happen if a task is scheduled to a different GPU than the expected one?

File: src/iceberg/executor/penguins.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        self._data_input_path = input_path
+        self._output_path = output_path
+        self._model_name = model
+        self._model_path = model_path
+        self._epoch = epoch
+        self._req_modules = None
+        self._pre_execs = None
+        self._env_var = os.environ.get('VE_PENGUINS')
+        if self._res_dict['resource'] == 'xsede.bridges':
```
Comment (iparask) [inline src/iceberg/executor/penguins.py:48]: I would suggest you use a `try except` block here or introduce an `else` to account for a resource other than Bridges.

**Review by iparask — CHANGES_REQUESTED** [review]
I agree with the proposed approach. I have left two comments in the code. Also would you mind fixing the linters at least for your work? We should open a ticket for the rest.

**Review by iparask — APPROVED** [review]

**Pull request merged**